action.yml

```diff
@@ -0,0 +1,119 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Create Cloud Deploy Release
+author: Google LLC
+description: |-
+  Use this action to create a release in Cloud Deploy
+
+inputs:
+  name:
+    description: |-
+      The name of the release.
+    required: true
+
+  delivery_pipeline:
+    description: |-
+      The delivery pipeline to use for the release.
+    required: true
+
+  region:
+    description: |-
+      Region in which the delivery pipeline can be found.
+    required: false
+
+  source:
+    description: |-
+      The location of the files to be included in the release.
+    required: false
+
+  build_artifacts:
+    description: |-
+      Path to a Skaffold output file containing the details of the image(s) to be released.
+    required: false
+
+  images:
+    description: |-
+      The images to be released
+    required: false
+
+  disable_initial_rollout:
+    description: |-
+      Prevent the release being deployed to the first target in the delivery pipeline.
+    default: false
+    required: false
+
+  gcs_source_staging_dir:
+    description: |-
+      A directory in Google Cloud Storage to copy the source used for staging the build.
+    required: false
+
+  skaffold_file:
+    description: |-
+      Path of the skaffold file absolute or relative to the source directory.
+    required: false
+
+  annotations:
+    description: |-
+      Add additional annotations to the release
```

Comment:
Nit: some of these descriptions end in a period and some do not. Please end all with a period.